### PR TITLE
Merge master into foxy

### PR DESCRIPTION
**Pull request by lukicdarkoo**
no conflicts

### CODE REVIEW

File: webots_ros2_control/CHANGELOG.rst [hunk 1]
```diff
@@ -0,0 +1,7 @@
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Changelog for package webots_ros2_control
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+1.0.6 (2021-05-21)
```
Comment (stefaniapedrazzi) [inline webots_ros2_control/CHANGELOG.rst:5]: Just a question, is the version `1.0.6` correct here?
If yes, shouldn't we add a line for version `1.1.0`?

File: webots_ros2_mavic/CHANGELOG.rst [hunk 2]
```diff
@@ -0,0 +1,7 @@
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Changelog for package webots_ros2_mavic
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+1.1.0 (2021-xx-xx)
```
Comment (stefaniapedrazzi) [inline webots_ros2_mavic/CHANGELOG.rst:5]: Here it seems that the date is missing.
This should be fixed in the master branch first, is it correct?

Comment (lukicdarkoo) [reply webots_ros2_mavic/CHANGELOG.rst:5]: Yes, I started a PR here:
https://github.com/cyberbotics/webots_ros2/pull/251

File: webots_ros2_control/CHANGELOG.rst (hunk 1 above)
Comment (lukicdarkoo) [reply webots_ros2_control/CHANGELOG.rst:5]: You are right, I opened a PR for it

**Review by stefaniapedrazzi — APPROVED** [review]
Thank you

**Pull request merged**